samples/XCT.Sample/ViewModels/Markup/SearchViewModel.cs

```diff
@@ -54,17 +52,24 @@ public SearchViewModel()
 					},
 				}
 			};
+
+			BackCommand = new RelayCommand(Back);
+			LikeCommand = new RelayCommand<Tweet>(Like);
+			OpenTwitterSearchCommand = new RelayCommandAsync(OpenTwitterSearch);
+			OpenHelpCommand = new RelayCommandAsync(OpenHelp);
 		}
 
-		public ICommand BackCommand => backCommand ??= new RelayCommand(Back);
+		public ICommand BackCommand { get; }
 
-		public ICommand LikeCommand => likeCommand ??= new RelayCommand<Tweet>(Like);
+		public ICommand LikeCommand { get; }
 
-		public ICommand OpenTwitterSearchCommand => openTwitterSearchCommand ??= new RelayCommandAsync(OpenTwitterSearch);
+		public ICommand OpenTwitterSearchCommand { get; }
 
-		public ICommand OpenHelpCommand => openHelpCommand ??= new RelayCommandAsync(OpenHelp);
+		public ICommand OpenHelpCommand { get; }
 
-		void Back() { }
+		void Back()
+		{
+		}
```

Comment:
can you revert this change, please?

Reply:
This is style related change. There was a warning for this `void Back() { }`

Reply:
Odd I can remember that's our pattern, or am I just think about another project? Well, so you can ignore this one.

Reply:
Reverted the change